mm2src/coins_activation/src/l2/init_l2.rs

```
@@ -205,10 +204,9 @@ where
         )
         .await?;
 
-        lp_register_coin(&self.ctx, coin.into(), RegisterCoinParams {
-            ticker: self.ticker.clone(),
-        })
-        .await?;
+        if let Ok(c_ctx) = CoinsContext::from_ctx(&self.ctx) {
+            c_ctx.add_l2(coin.into()).await.ok();
```

Please return an error from here instead of using `ok()`